content/cv/conv_and_padding.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Convolution & Padding"
+date: 2019-10-17T13:34:08+07:00
+draft: false
+authors: ["khoatranrb"]
+categories:
+  - computer vision
+tags:
+  - introductory
+  - convolution
+  - padding
+slug: /home/khoa/Desktop/vietcs-blog/content/cv/conv_and_padding.md
+---
+
+Convolution là kỹ thuật quan trọng trong *xử lý ảnh* (**Image processing**), được sử dụng chính yếu trong các phép toán trên ảnh như: đạo hàm ảnh (**gradient image**), làm trơn ảnh (**blurring**), nhận diện viền (**edge detection**), trích xuất đặc trưng (**feature extraction**).
+
+Bài viết này cung cấp cho bạn kiến thức cơ bản về *Convolution* và tính chất của nó.
+
+### **1. Giới thiệu về toán tử  (hệ thống) tuyến tính**
```

Comment:
Đã dùng # ## hay ### thì không cần thêm format in đậm, em sửa hết mấy cái còn lại nha. 
Ví dụ dùng header:
"1." hay "2." hay "3." là header bậc 1 dùng #
"1.1" hay "1.2" hay "2.1" là header bậc 2 dùng ##
"1.1.1" hay "1.1.2" hay "2.1.2" là header bậc 3 dùng ###
Em sửa lại hết mấy chỗ này nha. Đã cập nhật doc cho phần này